src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Or just add another prop to this class and then have the data get passed through [via `index.js` in this function](https://github.com/calvinclaus/react-event-calendar/blob/353c9e0f09ed349d67f988a26d806f22985ef523/src/index.js#L190)